Google.PowerShell/Sql/GcSqlInstanceCmdlets.cs

```diff
@@ -1388,4 +1388,159 @@ private Settings PopulateSetting(Settings newSettings)
             return newSettings;
         }
     }
+
+    /// <summary>
+    /// <para type="synopsis">
+    /// Failover a Cloud SQL Instance.
+    /// </para>
+    /// <para type="description">
+    /// Failover the specified Cloud SQL Instance to its failover replica instance.
+    /// </para>
+    /// <para type="description">
+    /// If a Project is specified, it will failover the specified Instance in that Project. Otherwise, failsover the 
+    /// Instance in the Cloud SDK config project. 
+    /// </para>
+    /// <example>
+    ///   <para>Failover the SQL Instance "test1" in the Project "testing."</para>
+    ///   <para><code>PS C:\> Failover-GcSqlReplica -Project "testing" -Instance "test1"</code></para>
+    ///   <br></br>
+    ///   <para>(If successful, the command returns nothing.)</para>
+    /// </example>
+    /// <example>
+    ///   <para>Failover the SQL Instance "test1" with current settings version 3 in the Project "testing."</para>
+    ///   <para>
+    ///     <code>PS C:\> Failover-GcSqlReplica -Project "testing" -Instance "test1" - SettingsVersion 3</code>
+    ///   </para>
+    ///   <br></br>
+    ///   <para>(If successful, the command returns nothing.)</para>
+    /// </example>
+    /// </summary>
+    [Cmdlet("Failover", "GcSqlInstance")]
+    public class FailoverGcSqlInstanceCmdlet : GcSqlCmdlet
+    {
+        private class ParameterSetNames
+        {
+            public const string ByName = "ByName";
+            public const string ByInstance = "ByInstance";
+        }
+
+        private class ErrorCodes
+        {
+            public const int InvalidSettingsVersion = 412;
+        }
+
+        private class ErrorMessages
+        {
+            public const string InvalidSettingsVersion =
+                "Input or retrieved settings version does not match current settings version for this instance.";
+        }
+
+        /// <summary>
+        /// <para type="description">
+        /// Name of the project in which the Instance resides.
+        /// Defaults to the Cloud SDK config for properties if not specified.
```

Comment:
Just that it looks in the properties of the config for the project. I can change it to just say "the Cloud SDK project" though.